contracts/digg-core/UFragments.sol

```diff
@@ -302,4 +306,29 @@ contract UFragments is ERC20Detailed, Ownable {
         emit Approval(msg.sender, spender, _allowedFragments[msg.sender][spender]);
         return true;
     }
+
+    /**
+     * @notice Mints the reimbursement for remDIGG one time, directly to dev multisig
+     * @dev This is implemented to address BIP92 (https://forum.badger.finance/t/bip-92-digg-restructuring-v3-revised/5653)
+     * @dev by allowing the development multisig a one time mint of the totalSupply of remDIGG for distribution
+     */
+    function oneTimeMint() external onlyOwner {
+        require(!remDiggMint, "Mint already complete");
+        uint256 mintAmount = 52942035500;
+        uint256 shareValue = mintAmount.mul(_sharesPerFragment);
+        _shareBalances[0xB65cef03b9B89f99517643226d76e286ee999e77] = _shareBalances[0xB65cef03b9B89f99517643226d76e286ee999e77].add(shareValue);
```

Comment:
Better to store the address as a `constant` since it's used multiple times. Private constants don't add any overhead since they're replaced during compilation